### ISSUE TITLE
Missing aria-label on pagination <nav> element

### ISSUE THREAD

**Issue by xi** [zostera/django-bootstrap5#449]
Since #60, pagination is wrapped in a `<nav>` element. This is a good thing because screen reader users can find the pagination quickly via a landmark. But it can also cause confusion because it is not clear which navigation landmark is which. The  [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#generalprinciplesoflandmarkdesign) therefore recommends to add labels:

> If a specific landmark role is used more than once on a page, provide each instance of that landmark with a unique label.

I see 3 options how to fix this:

1. Provide a fixed label (e.g. "Pagination") along with translations
2. Add a `label` parameter to the  `bootstrap_pagination` template tag
3. Remove the `<nav>` element from django-bootstrap5 and tell authors to add it themselves

Which option do you think works best?
